internal/config/config.go

```diff
@@ -63,7 +64,7 @@ type Settings struct {
 func defaultConfig(dataDir, configFile string) *Config {
 	forcePrepend := forcePrependDefault
 	forcePrependEnv := os.Getenv("ASDF_FORCE_PREPEND")
-	if forcePrependEnv == "yes" || (forcePrependEnv == "" && runtime.GOOS == "darwin") {
+	if matched, _ := regexp.MatchString(`yes|YES`, forcePrependEnv); matched || (forcePrependEnv == "" && runtime.GOOS == "darwin") {
```

Comment:
Using regex for this is overkill. Suggest using `strings.ToLower(forcePrependEnv) == "yes"` instead.

Reply:
@andrecloutier @Stratus3D 
Thank you for the comment and emoji reaction.
Yes, I know. I prefer a simple implementation too, so if I can do it that way, that's the way to go.

However, I'm concerned that it implementation might be a bit too loose since it would allow typo strings like “yEs” and “yeS” while “yes” and “YES” and “Yes” would be have consistency, but I'm not sure if that's ok.
Is that too much of a concern?

Reply:
I'm not concerned about that. If someone wants to specify odd casing that is fine with me. It will keep the implementation simpler.

Reply:
@Stratus3D 
Alright! I’ll do it that way. Thanks!

Reply:
@andrecloutier @Stratus3D 
Would you please code review again?